### DISCUSSION TITLE
Inescapable Emoji Handling an issue

### DISCUSSION THREAD

**Discussion by jabofh** [requarks/wiki#6975] [Error / Bug Report]
Hi all,

I'm busy documenting using Pacemaker to configure a HA cluster for my team. That as all fine, until I wanted to mention configuring the HeartBeat and got :heartbeat:...

I bypassed the emoji handling by in-line code quoting `ocf:heartbeat:named`; but you should probably be able to `\`-escape the first or second `:` to bypass the emoji processing?

**Comment by fnafox** [comment]
You can escape it by placing it into a span element, like so `<span>:</span>` but I agree that you should be able to escape it in a more intuitive fashion.
